demo/test_device_inventory_v2.py

```diff
@@ -2,13 +2,24 @@
 import hashlib
 
 parent_dir = os.path.dirname
-APP_BIN_PATH = os.path.join(parent_dir(parent_dir(__file__)), 'cyences_app_for_splunk', 'bin')
+APP_BIN_PATH = os.path.join(parent_dir(parent_dir(__file__)), "cyences_app_for_splunk", "bin")
 sys.path.insert(0, APP_BIN_PATH)
 
 from device_inventory_v2_util import DeviceEntry, DeviceManager
+from cyences_device_manager import logger
+import splunk.auth as auth
+from splunk import rest
 
 
+# Following things needs to be configured as per the splunk environment
+SPLUNK_USERNAME = "admin"
+SPLUNK_PASSWORD = "admin123"
 HOSTNAME_POSTFIXES = ".ad.crossrealms.com, .crossrealms.com"
```

Comment:
Add new line above this as keep the SPLUNK_USERNAME and SPLUNK_PASSWORD those 2 are only user needs to configure, rest of the variables are static.